app/src/test/java/org/simple/clinic/contactpatient/ContactPatientInitTest.kt

```diff
@@ -37,7 +37,7 @@ class ContactPatientInitTest {
         .whenInit(defaultModel)
         .then(assertThatFirst(
             hasModel(defaultModel.contactPatientInfoLoading()),
-            hasEffects(LoadPatientProfile(patientUuid), LoadLatestOverdueAppointment(patientUuid))
+            hasEffects(LoadPatientProfile(patientUuid), LoadLatestOverdueAppointment(patientUuid), LoadCurrentFacility)
```

Comment:
Can we update the test name to indicate that we are testing loading current facility as well, please?